internal/config/config.go

```diff
@@ -251,9 +377,13 @@ func parsePeer(p peer) (*Peer, error) {
 	if ip == nil {
 		return nil, fmt.Errorf("invalid peer IP %q", p.Addr)
 	}
-	holdTime, err := parseHoldTime(p.HoldTime)
-	if err != nil {
-		return nil, err
+	holdTime := 90 * time.Second
+	if p.HoldTime != "" {
+		ht, err := ParseHoldTime(p.HoldTime)
+		if err != nil {
+			return nil, err
+		}
+		holdTime = ht
```

Comment:
Yup, I've done so following https://github.com/metallb/metallb/pull/593#discussion_r439627115. Rather than having a hardcoded default inside the function, we set the hardcoded default in the calling function, which allows us to reuse `ParseHoldTime()` in `bgp_controller.go`.

>Any reason to not do it in a separate commit?

One reason is that I've been asked to reduce the number of commits in this PR... I can move it if you want but let's focus on getting this PR to a merge.

Reply:
Yes, please. That is a very weak reason. The previous PR had lot of commits because they were all the intermediate states that you worked on. Having clean separate commits helps to review :)

Reply:
Moved to a separate commit.